apisix/admin/services.lua

```diff
@@ -67,18 +66,6 @@ local function check_conf(id, conf, need_id, schema)
         end
     end
 
-    if conf.script then
-        local obj, err = loadstring(conf.script)
-        if not obj then
-            return nil, {error_msg = "failed to load 'script' string: "
-                                     .. err}
-        end
-
-        if type(obj()) ~= "table" then
-            return nil, {error_msg = "'script' should be a Lua object"}
-        end
-    end
-
```

Comment:
It's just a validation for `script`. 
Please check the file `apisix/script.lua`